tests/integrate/tools/catch_properties.sh

```diff
@@ -65,7 +65,8 @@ test -e $1 && rm $1
 #--------------------------------------------
 if [ $calculation != "nscf" ] && [ $calculation != "get_wf" ]\
 && [ $calculation != "get_pchg" ] && [ $calculation != "get_S" ]; then
-	etot=`grep ETOT_ $running_path | awk '{print $2}'`
+	#etot=`grep ETOT_ $running_path | awk '{print $2}'` 
+	etot=$(grep "ETOT_" "$running_path" | tail -1 | awk '{print $2}')
```

Comment:
ETOT_ output should be once. which code will cause more outputs?

Reply:
If you run the md_restart test case a second time on your computer and forget to delete the OUT.autotest/running_md.log file, the file will contain the logs from both the first and second runs, where there are two ETOT_.